### PR TITLE
Fix extra line in Fenced Github Flavored Codeblocks

### PR DESCRIPTION
**Pull request by RickStrahl**
This fixes the extra line in Fenced code blocks by trimming the HTML content for linefeeds/carriage returns.

This relates to Issue #35

### CODE REVIEW

**Comment by RickStrahl** [comment]
Apologies for the reformatted test file - didn't notice until it got submitted here.

May I suggest using an .editorConfig to minimize editor setting differences across platform?

**Comment by mysticmind** [comment]
@RickStrahl sure, we can add a.editorConfig, it looks to be essential. Got caught up with my day job, looking at this PR now.

File: src/ReverseMarkdown/Converters/Pre.cs [hunk 1]
```diff
@@ -17,7 +17,7 @@ public override string Convert(HtmlNode node)
             if (Converter.Config.GithubFlavored)
             {
                 var lang = GetLanguage(node);
-                var code = DecodeHtml(node.InnerText);
+                var code = DecodeHtml(node.InnerText).TrimEnd(new[] {'\r', '\n'});
```
Comment (mysticmind) [inline src/ReverseMarkdown/Converters/Pre.cs]: Test is failing on TravisCI(*nix), changing `.TrimEnd(new[] {'\r', '\n'})` to `TrimEnd('\r').TrimEnd('\n')` should take care of it.

**Comment by RickStrahl** [comment]
So it looks like the problem isn't the TrimEnd() but rather the test itself and the embedded linefeed/carriage conversion which behaves different on the two platforms. 

I've changed the test to explicitly add the {System.Environment.NewLine} which fixed the issue, but this would also be addressed by adding the `.editorconfig` as it will keep the linefeeds consistent (ie. take my Windows linefeeds and turn them into \n which is the default in .editorconfig).

Anyway it builds now with the original `.TrimEnd(new [] { '\n', '\r' })` code.

**Comment by mysticmind** [comment]
@RickStrahl 3.5.0 was released to NuGet earlier today.

**Pull request merged**